solutions/USACO dec16-moocast-silver.mdx

```diff
@@ -111,4 +111,78 @@ print(ans)
 
 </PySection>
 
+<JavaSection>
+
+```java
+import java.io.File;
+import java.io.PrintWriter;
+import java.util.Scanner;
```

Comment:
> import java.io.File;
import java.io.PrintWriter;
import java.util.Scanner;

import java.io.*;
import java.util.*;
is a lot cleaner

Reply:
`import java.io.*;
import java.util.*;
`

some reason the star got deleted :(

Reply:
Am using `*` now.